### DISCUSSION TITLE
به فایل libv2ray.arrنیاز دارم

### DISCUSSION THREAD

**Discussion by mahxray** [GFW-knocker/Segaro_Dream#12] [General]
سلام سگارو جان اگر امکانش هست فایلشو بهم بده هرکاری کردم نتونستم بیلدش کنم

**Comment by GFW-knocker** [comment]
> سلام سگارو جان اگر امکانش هست فایلشو بهم بده هرکاری کردم نتونستم بیلدش کنم

اخرین نسخه فایل arr اینجاست
https://github.com/yuhan6665/v2rayNG/releases/download/aar-test/libv2ray.aar
https://github.com/yuhan6665/v2rayNG/releases/tag/aar-test

دوست عزیزمون سگارو ایشون هست و بنده از مریدانشونم فقط
https://github.com/iSegaro
https://twitter.com/isegaro
